shell/platform/darwin/macos/framework/Source/MockFlutterViewController.h

```diff
@@ -0,0 +1,31 @@
+#import <Foundation/NSString.h>
+#import <OCMock/OCMock.h>
+
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterDartProject_Internal.h"
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterViewController_Internal.h"
+#import "flutter/testing/testing.h"
+
+namespace flutter::testing {
+
+// Returns a mock FlutterViewController that is able to work in environments
+// without a real pasteboard.
+static id CreateMockViewController(NSString* pasteboardString) {
```

Comment:
The file name `MockFlutterViewController` made me think this is a new class.
After looking at the code I realized it is a utility method to create an OCMock for a `FlutterViewController`.

I think we can do one of the following instead:
1. Rename the file as "FlutterViewControllerTestsUtils.h`, also add a `FlutterViewControllerTestsUtils.mm` for the implementation.
2. Directly adding this method to `FlutterViewControllerTest.mm` 

I'm OK with either approach above, if there are more than 1 util methods used in `FlutterViewControllerTest.mm` , I'd go with option 1.

Reply:
Went with approach 1 for now.